docs/guide/middleware.md

```diff
@@ -34,3 +34,53 @@ export default {
   auth: 'guest'
 }
 ```
+
+## Accessing user data
+
+With the user endpoints provided in some of the schemes you may access additional information in your middleware using `this.$auth.user`.
+
+For example if the user endpoint provides additional information like so, you can 
+
+```json
+{
+    "user":{
+      "is_superadmin":true,
+      "is_staff":true,
+    }
+}
+```
+
+You could now create a new middleware file such as `is_superadmin.js` with the following content:
+
+```js
+export default function({$auth,redirect}){
```

Comment:
Can you change intent space of the code examples to 2 and add spaces after function, if, and in brackets please? :)

Reply:
like so 

```js
export default function({
  $auth,
  redirect
}) {
  if ($auth.user.is_superadmin === false) {
    redirect('/')
  }
}
```

Reply:
I mean something like this `export default function ({ $auth, redirect })`

Reply:
ok will change. I need to configure my IDE to reflect the style